src/Makefile.test.include

```diff
@@ -219,6 +219,7 @@ endif
 if ENABLE_FUZZ_BINARY
 test_fuzz_fuzz_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
 test_fuzz_fuzz_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+test_fuzz_fuzz_CFLAGS = $(PIE_FLAGS)
```

Comment:
Presumably we should do some cargo culting by adding

    test_fuzz_fuzz_CFLAGS = $(AM_CFLAGS) $(PIE_FLAGS)

Reply:
Ah, yeah, good idea. I will copy this on the next force-push.